packages/rest/src/controllers/didcomm/out-of-band/OutOfBandController.ts

```diff
@@ -0,0 +1,266 @@
+import type { DidCommOutOfBandRecord } from './OutOfBandControllerTypes'
+import type { PlaintextMessage } from '@credo-ts/core/build/types'
+
+import {
+  AgentMessage,
+  JsonTransformer,
+  OutOfBandInvitation,
+  RecordNotFoundError,
+  ConnectionInvitationMessage,
+} from '@credo-ts/core'
+import { parseMessageType, supportsIncomingMessageType } from '@credo-ts/core/build/utils/messageType'
```

Comment:
Should we export these or is this quite niche? Same as with the PlaintextMessage type?

Reply:
Yes we should probably export these